src/pages/brand.tsx

```diff
@@ -0,0 +1,446 @@
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section>
+        <h2>Color &amp; Typeface</h2>
+        <p>Our core color is called Rhodamine.</p>
+        <p>The GraphQL wordmark and headlines are set in Rubik Light (300).</p>
+        <div className="samples">
+          <div className="rhodamine-sample">
+            Rhodamine
+            <dl>
+              <dt>P3</dt>
+              <dd>0.8824, 0, 0.5961</dd>
+              <dt>RGB</dt>
+              <dd>225, 0, 152 / #E10098</dd>
+              <dt>PMS</dt>
+              <dd>
+                <a
+                  href="https://www.pantone.com/color-finder/Rhodamine-Red-C"
+                  target="_blank"
+                >
+                  Rhodamine Red C
+                </a>
+              </dd>
+              <dt>CMYK</dt>
+              <dd>5, 92, 0, 0</dd>
+            </dl>
+          </div>
+          <div className="rubik-sample">
+            <a
+              href="https://fonts.google.com/specimen/Rubik?sidebar.open=true&selection.family=Rubik:wght@300&preview.text=GraphQL&preview.text_type=custom"
+              target="_blank"
+            >
+              Rubik Light
+            </a>
+            <dl>
+              <dt>Weight</dt>
+              <dd>300</dd>
+              <dt>License</dt>
+              <dd>
+                <a
+                  href="https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL"
+                  target="_blank"
+                >
+                  Open Font License
+                </a>
+              </dd>
+            </dl>
+            <div className="sample">GraphQL &Aring;BC&oslash;rm123</div>
+          </div>
+        </div>
+        <div className="dos-donts">
+          <ul className="do">
+            <li>
+              <em>Do</em> use Rhodamine alongside grayscale or
+              neutral&nbsp;colors.
+            </li>
+            <li>
+              <em>Do</em> prefer the P3 wide gamut color over RGB in digital;
+              the PMS spot color over CMYK process in&nbsp;print.
+            </li>
+            <li>
+              <em>Do</em> use Rubik Light with optical kerning and standard
+              letter&nbsp;spacing.
+            </li>
+          </ul>
+          <ul className="dont">
+            <li>
+              <em>Don't</em> use Rhodamine for large background&nbsp;areas.
+            </li>
+            <li>
+              <em>Don't</em> directly combine Rhodamine with other bright or
+              saturated&nbsp;colors.
+            </li>
+            <li>
+              <em>Don't</em> bold, italicize, or alter the letter spacing of
+              the&nbsp;GraphQL&nbsp;wordmark.
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section>
+        <h2>Logo &amp; Hexagraph</h2>
+        <p>
+          Our logomark is called the Hexagraph. The GraphQL logo is composed of
+          the hexagraph and the GraphQL wordmark set in Rubik Light.
```

Comment:
Capitalisation consistency: `Hexagraph` or `hexagraph`? I went lower case as it's commonly used in lower case further down the page, and used quotes to make the definition clearer instead of the capital.

```suggestion
          Our logomark is called the “hexagraph”. The GraphQL logo is composed of
          the hexagraph and the GraphQL wordmark set in Rubik Light.
```